src/type/definition.js

```diff
@@ -585,6 +585,18 @@ export class GraphQLScalarType {
   inspect: () => string;
 }
 
+Object.defineProperty(GraphQLScalarType, Symbol.toStringTag, {
+  get() {
+    return this.name;
+  },
+});
+
+Object.defineProperty(GraphQLScalarType.prototype, Symbol.toStringTag, {
+  get() {
+    return this.constructor.name;
+  },
+});
```

Comment:
I'm noticing this implementation is identical for all types. Perhaps a function in jsutils to reduce code duplication?